mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -1755,7 +1755,7 @@ def search_logged_models(
             models = (
                 session.query(SqlLoggedModel)
                 .filter(SqlLoggedModel.experiment_id.in_(experiment_ids))
-                .order_by(SqlLoggedModel.creation_timestamp_ms)
+                .order_by(SqlLoggedModel.creation_timestamp_ms.desc())
```

Comment:
`SqlAlchemyStore.search_logged_models` is still incomplete. We have a ticket to complete it.